docs/guides/getting-started/getting-started-kube.adoc

```diff
@@ -41,6 +41,8 @@ kubectl create -f https://raw.githubusercontent.com/keycloak/keycloak-quickstart
 This command starts {project_name} on Kubernetes and creates an initial admin user with the username `admin` and password
 `admin`.
 
+NOTE: if you will eventually want more than 1 server replica please see <<getting-started-scaling-and-tuning.adoc#horizontal-scaling,Horizontal Scaling>>.
```

Comment:
> Hi, please re-write the link to use Freemarker to make it work in downstream.

No problem. When I saw that getting started wasn't in the links.adoc, and a link of this format in user-federation.adoc, I just assumed this could be used as a one off. I'll update that link as well.